### ISSUE TITLE
Parent defined rules not working in child forms

### ISSUE THREAD

**Issue by gdiazderadaa** [jlorente/yii2-activerecord-inheritance#4]
Hi there,
Are the parent rules supposed to be inherited by the child class? I'm assuming so since the properties theirselves are. However that's not working for me.
Thanks

**Comment by jlorente** [comment]
Hello!
I saw no need to inherit the parent rules, because the save and validation methods of the child class call the methods of the parent class, so the parent rules affect the parent properties already and the getErrors method return all the errors in the two classes when it is called.

**Comment by gdiazderadaa** [comment]
Hello!
Then I may be missing something, because the client-side validation of the child class doesn't work for inherited properties if the rules are setup in the parent class...

**Comment by jlorente** [comment]
You are right in that. The client side validation doesn't work, because the parent rules are not inherited by default. I will think about it and fix the issue.

**Comment by mandachuvamg** [comment]
Hi!

Has any solution been defined for this problem?

Force the merge with the parent is the best solution?

Thanks!
